### PR TITLE
Bk 723 create init request

### PR DESCRIPTION
**Pull request by mrroot5**
https://doofinder.atlassian.net/browse/BK-723

### CODE REVIEW

File: pydoof/search_api/enums.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from enum import Enum, unique
```
Comment (sonic182) [inline pydoof/search_api/enums.py:1]: I think you added this file by mistake

**Review by sonic182 — CHANGES_REQUESTED** [review]

File: pydoof/search_api/enums.py (hunk 1 above)
Comment (mrroot5) [reply pydoof/search_api/enums.py:1]: Yep, probablly in a merge

**Review by sonic182 — APPROVED** [review]

**Review by vpoten — APPROVED** [review]

File: tests/search_api/test_stats.py [hunk 2]
```diff
@@ -6,6 +6,18 @@
 
 class TestStats(unittest.TestCase):
 
+    @mock.patch('pydoof.search_api.stats.SearchAPIClient')
+    def test_init_session(self, APIClientMock):
```
Comment (GLagunilla) [inline tests/search_api/test_stats.py:10]: I'm not sure how the tests works here, but... Should it be a tests for each flux? `Ok`, `Search engine not found`, `Bad request`,...

Comment (mdogo) [reply tests/search_api/test_stats.py:10]: Not really, this only tests that the client is called with the right parameters and URL. There is a different suit of tests that check that each Doophoenix response has the proper function return.

**Review by mdogo — APPROVED** [review]

**Review by GLagunilla — APPROVED** [review]

**Pull request merged**